### PR TITLE
Staticcheck added to workflow

### PR DESCRIPTION
**Pull request by mobilarte**
This adds job `Staticcheck` in addition to `Format` and `Vet` to the workflow.

### CODE REVIEW

**Comment by vapourismo** [comment]
Hey, thanks for the contribution. I will review this in about 2 weeks. I am currently away.

File: knx/cemi/ldata_test.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ func makeRandBuffer(n int) []byte {
 	return buffer
 }
 
+/*
```
Comment (vapourismo) [inline knx/cemi/ldata_test.go:19]: Is this comment added because the function is not used? If so, please remove the function entirely.

**Review by vapourismo — CHANGES_REQUESTED** [review]
Looks good. Just one change to be made.

File: knx/cemi/ldata_test.go (hunk 1 above)
Comment (mobilarte) [reply knx/cemi/ldata_test.go]: Commented code has been removed. I could also push the changes to `go.mod` and `go.sum` after running `go mod tidy`. The scheduled action could also be changed to twice a month, there is no real need to run it once a day. If ok I will push the modifications into this PR.

**Review by vapourismo — APPROVED** [review]

**Pull request merged**